src/ui/MLFacet/MLFacetValues/MLFacetValues.ts

```diff
@@ -38,16 +45,20 @@ export class MLFacetValues {
     return this.facetValues.filter(value => value.isSelected).map(value => value.value);
   }
 
-  public hasSelectedValues() {
+  public get hasSelectedValues() {
     return !!findWhere(this.facetValues, { state: FacetValueState.selected });
   }
 
   public clearAll() {
     this.facetValues.forEach(value => value.deselect());
   }
 
-  public isEmpty() {
-    return !this.facetValues.length;
+  public get isEmpty() {
+    return !this.length;
+  }
+
+  public get isExpanded() {
+    return this.length > this.facet.options.numberOfValues;
```

Comment:
I think it would be good to add a getter on the facet class to encapsulate the second part of the condition:

```
public get configuredNumberOfValues() {
    return this.options.numberOfValues;
}
```

Reply:
I don't really agree, the options object is already accessible and used in multiple places. I don't want to add a getter for every options that is used it will make the heavy MLFacet class even more heavy.